rule-audit-client/src/services/Capi.ts

```diff
@@ -0,0 +1,77 @@
+import { urls } from "../constants";
+
+export type CapiResponse<TContent> = {
+  status: "ok" | "error";
+  total: number;
+  startIndex: number;
+  pageSize: number;
+  currentPage: number;
+  pages: number;
+  results: TContent[] | undefined;
+};
+
+export type CapiContentResponse = CapiResponse<CapiContent>;
+
+export type CapiTagsResponse = CapiResponse<CapiTag>;
+
+export type CapiSectionsResponse = CapiResponse<CapiSection>;
+
+export type CapiContent = {
```

Comment:
IIRC @alexduf was working on adding TS types to CAPI, maybe we can use them here?

Reply:
Absolutely! I don't know what the status of this project is, but I'm very keen to use them as soon as they're available.